projects/1.19/assets/random-enchants-2/randomenchants2/lang/zh_cn.json

```
@@ -0,0 +1,94 @@
+{
+  "enchantment.randomenchants2.solar_enchant": "日光修补",
+  "enchantment.randomenchants2.solar_enchant.desc" : "放在主副手上的装备在光源附近下会缓慢恢复",

```

```suggestion
  "enchantment.randomenchants2.solar_enchant.desc" : "放在主副手上的物品在光源附近或阳光下会缓慢修复",
```
原文如此（甚至没有`缓慢`）
原文只提到daytime，请问雷雨天或者下界/末地会修补吗